src/NServiceBus.Core/Licensing/LicenseDetailsProvider.cs

```diff
@@ -0,0 +1,20 @@
+﻿namespace NServiceBus.Features
+{
+    class LicenseDetailsProvider : ILicenseDetailsProvider
+    {
+        public string LicenseId { get; }
+        public string CustomerName { get; }
+
+        public LicenseDetailsProvider(string licenseId, string customerName)
+        {
+            LicenseId = licenseId;
+            CustomerName = customerName;
+        }
+    }
+
+    interface ILicenseDetailsProvider
```

Comment:
Does the interface give us any value or could we just use  a `LicenseDetails` DTO instead?